docs/configuration.rst

```diff
@@ -200,3 +173,34 @@ These are settings that are specific to dachar::
     concern_threshold = 0.2
     # possible locations for scans and analysis of datasets
     locations = ceda dkrz other
+
+
+catalog maker
```

Comment:
would this not be part of the `catalog-maker` documentation?

Reply:
@cehbrecht  For all the other packages, I have been putting a reference to this page in their docs, see https://daops.readthedocs.io/en/latest/usage.html, so plan to do that here as well. I put it in here because it references the structure of the roocs-utils config and the `config_data_types` section is explained. 

But also happy to move it out if you think it makes more sense!

Reply:
@ellesmith88 Ok ... it is a shared documentation. I thought the `catalog_maker` is an independent component (and as such the config). Maybe `roocs_utils` would provide the config tools ... config options are part of the other components. But that could be another topic :) So keep it as it is :)